client/jetpack-cloud/sections/partner-portal/payment-methods-v2/index.tsx

```diff
@@ -1,10 +1,11 @@
+import { Button } from '@automattic/components';
 import { useTranslate } from 'i18n-calypso';
-import Layout from 'calypso/jetpack-cloud/components/layout';
-import LayoutHeader, {
-	LayoutHeaderSubtitle as Subtitle,
-	LayoutHeaderTitle as Title,
-} from 'calypso/jetpack-cloud/components/layout/header';
-import LayoutTop from 'calypso/jetpack-cloud/components/layout/top';
+import CardHeading from 'calypso/components/card-heading';
+import Layout from '../layout';
+import LayoutHeader from '../layout/header';
+import LayoutTop from '../layout/top';
```

Comment:
I suggest using the layout-related component from `jetpack-cloud/components` since it was designed to be reused throughout the Jetpack Cloud.

Reply:
Thanks for the heads up. I was reviewing the other `Purchases` pages and noticed the previous implementations so I went with `partner-portal` for consistency but this change makes sense.